app/gui/src/dashboard/components/dashboard/DirectoryNameColumn.tsx

```diff
@@ -48,7 +42,7 @@ export default function DirectoryNameColumn(props: DirectoryNameColumnProps) {
     storeState.expandedDirectoryIds.includes(item.id),
   )
 
-  const updateDirectoryMutation = useMutation(backendMutationOptions(backend, 'updateDirectory'))
+  const updateDirectoryMutation = useBackendMutation(backend, 'updateDirectory')
```

Comment:
What was the purpose of moving away from the options?

Reply:
it still uses options internally, this *was* just a wrapper so we can arbitrarily wrap mutations. although i guess we can do that with options on the `MutationCache` itself, and previously the `useMemo` was there but it's no longer necessary because of React Compiler